### PR TITLE
[HTML] DOCTYPE should be case-insensitive

### PR DESCRIPTION
**Pull request by jfcherng**
Fixes https://github.com/sublimehq/Packages/issues/2236.

### CODE REVIEW

**Comment by deathaxe** [comment]
According to whatwg.org the `(?:PUBLIC|SYSTEM)\b` should be case-insensitive, too.

**Comment by jfcherng** [comment]
@deathaxe nice catch. done.

File: HTML/HTML.sublime-syntax [hunk 1]
```diff
@@ -280,7 +280,7 @@ contexts:
     - include: else-pop
 
   doctype-content-type:
-    - match: (?:PUBLIC|SYSTEM)\b
+    - match: (?i)(?:PUBLIC|SYSTEM)\b
```
Comment (deathaxe) [inline HTML/HTML.sublime-syntax]: Why not the shorter `(?i:PUBLIC|SYSTEM)\b` ??

Comment (jfcherng) [reply HTML/HTML.sublime-syntax]: for coding style

Comment (deathaxe) [reply HTML/HTML.sublime-syntax]: Most tags use the `(?i: ...)` style.

Comment (deathaxe) [reply HTML/HTML.sublime-syntax]: ... ok. A bit a mess.

Comment (jfcherng) [reply HTML/HTML.sublime-syntax]: changed to `(?i: ...)` style

**Comment by jfcherng** [comment]
Moved to https://github.com/sublimehq/Packages/pull/2239

**Pull request closed without merging**